### ISSUE TITLE
Failing to download chart when applying service plan

### ISSUE THREAD

**Issue by mattsmithies** [Tymlez/guardian-terraform#4]
## Overview

We have pulled down the latest code from this repository and forked it, so far, we have the infrastructure running, but when we are trying to deploy the services through terraform we are receiving "could not download chart: failed to download" 

## Notes:

1. I'm running Terraform v1.2.7
2. We copied the **vars.auto.tfvars_sample** file and added sensible defaults
3. I uncommented all the references to helm, as terraform was expecting params in the CLI

Below is an output of the errors on display.

```
Babayaga:services mattsmithies$ terraform apply services.plan
module.helm-charts.helm_release.mongodb: Creating...
module.helm-charts.helm_release.ingress-nginx[0]: Creating...
module.helm-charts.helm_release.guardian-message-broker: Creating...
╷
│ Error: could not download chart: failed to download "https://charts.bitnami.com/bitnami/mongodb-10.30.0.tgz" at version "10.30.0"
│ 
│   with module.helm-charts.helm_release.mongodb,
│   on modules/helm-charts/helm-charts.tf line 6, in resource "helm_release" "mongodb":
│    6: resource "helm_release" "mongodb" {
│ 
╵
╷
│ Error: could not download chart: failed to download "https://github.com/nats-io/k8s/releases/download/nats-0.17.5/nats-0.17.5.tgz"
│ 
│   with module.helm-charts.helm_release.guardian-message-broker,
│   on modules/helm-charts/helm-charts.tf line 64, in resource "helm_release" "guardian-message-broker":
│   64: resource "helm_release" "guardian-message-broker" {
│ 
╵
╷
│ Error: could not download chart: failed to download "https://github.com/kubernetes/ingress-nginx/releases/download/helm-chart-4.2.1/ingress-nginx-4.2.1.tgz" at version "4.2.1"
│ 
│   with module.helm-charts.helm_release.ingress-nginx[0],
│   on modules/helm-charts/helm-charts.tf line 483, in resource "helm_release" "ingress-nginx":
│  483: resource "helm_release" "ingress-nginx" {
│ 
╵

```

**Comment by mattsmithies** [comment]
@voycey Wanted to check if there was an obvious fix for this above problem we found.

**Comment by voycey** [comment]
Sorry mate I didn't get notified of this one, for the most part the helm provider only accepts OCI:// type URIs so it has to be an actual helm repository rather than a link to the chart tgz.

I'll look into this using those URLs and see if I can get it working with them directly though

**Comment by mattsmithies** [comment]
@voycey Ok, thanks for the reply. Understandably there is a need for the URI types, when Tymlez is deploying the Guardian are there any additional steps that aren't on the README?

Do you build and then upload the given Docker images to a OCI-compliant registry beforehand?

**Comment by voycey** [comment]
The images in the guardian come from the Envision repos, make sure you are on the latest version with the code in the repo (it's in the vars file).

I'm putting this in my calendar to look at tomorrow, will be online prior to the call if you want to go through it?

**Comment by mattsmithies** [comment]
Yes, happy to jump on beforehand to see how you get it running and to see if there are any insight infra nuggets I can steal.

**Comment by voycey** [comment]
Can you try this: https://github.com/Tymlez/guardian-terraform/releases/tag/v1.0.0b

Also have you made any changes inside helm_charts.tf? Not sure why its trying to download the specific versions as shown in your example
